.devcontainer/devcontainer.json

```diff
@@ -5,7 +5,10 @@
 			"version": "3.11"
 		}
 	},
-	"initializeCommand": "pip install poetry && poetry lock --no-update && poetry install",
+<<<<<<< HEAD
```

Comment:
Merge conflict code will cause the project to break. Make sure to resolve this before checking in.